src/visualization/components/Content.tsx

```diff
@@ -53,5 +59,9 @@ class Content extends React.Component<any, any> {
 // mapStateToProps and matchDispatchToProps using fat arrow function
 export default connect(
     state => { return { impression: state.impression, notfound: state.notfound, menu: state.menu, view: state.view } },
-    dispatch => { return bindActionCreators({ selectView: selectView }, dispatch) }
+    dispatch => { return bindActionCreators({ 
+        selectView: selectView, 
+        showSettleEvents: showSettleEvents
```

Comment:
>showSettleEvents [](start = 26, length = 16)

this doesn't actually do anything does it? onChange is still calling selectView

Reply:
The select view was there before, but the ShowSEttleEvents is indeed not useful here. 

---
In reply to: [175850712](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175850712) [](ancestors = 175850712)